docs/api/controller.md

```diff
@@ -109,20 +109,29 @@ By default, the node will be included securely (with encryption) if a network ke
 
 ### Managing associations
 
-The following methods can be used to manage associations between nodes. This only works AFTER the interview process!
+The following methods can be used to manage associations between nodes and/or endpoints. This only works AFTER the interview process!
 
 ```ts
-getAssociationGroups(nodeId: number): ReadonlyMap<number, AssociationGroup>;
-getAssociations(nodeId: number): ReadonlyMap<number, readonly Association[]>;
-isAssociationAllowed(nodeId: number, group: number, association: Association): boolean;
-addAssociations(nodeId: number, group: number, associations: Association[]): Promise<void>;
-removeAssociations(nodeId: number, group: number, associations: Association[]): Promise<void>;
+getAssociationGroups(nodeId: number, endpointIndex?: number): ReadonlyMap<number, AssociationGroup>;
+
+getAssociations(nodeId: number, endpointIndex?: number): ReadonlyMap<number, readonly AssociationAddress[]>;
```

Comment:
@AlCalzone Wouldn't it make sense to use `AssociationAddress` instead of two separete args?

Reply:
Changed and deprecated the old signature to be in sync with the other deprecations.